2016/proposals/singh-shubham.md

```diff
@@ -0,0 +1,210 @@
+
+![HTML MOCKUP flowchart](https://cloud.githubusercontent.com/assets/15982349/13770094/aa23d670-eaaa-11e5-81a8-312f860ecd7b.png)
+
+1. Interested Trainees apply as Group or Individuals by filling forms.
+![HTML MOCKUP FORMS](https://cloud.githubusercontent.com/assets/15982349/13816694/6c7a68e6-ebb5-11e5-809e-eb5bd00d7a22.png)
+2. All applications are listed in the *ALL_APPLICATIONS* view. In this view the lead trainer can accept or reject an application.
+The lead trainer has to compose one sample email by filling a web-form before accepting an application, this email will contain information such as
+date, location, mode(online or in person), instructor name and instructor email id. This information is also stored in the database for the accepted applicant to be queried on later.
+Accepting an application will send an email, to the applicant that he/she has been accepted into the program with other details about the actual training along with one randomly generated password to login in to their account (this password can be changed to anything by the trainee after the first login).
+The sample email has to be written once and can be sent to all the people who have been accepted for a particular training session.
+For another training session the lead trainer would just have to fill the web-form again, overwriting previous values.
+![HTML MOCKUP ALL_APPLICATIONS]( https://cloud.githubusercontent.com/assets/15982349/13816688/66a3e7e4-ebb5-11e5-83f1-5086ddddac13.png)
+
+3. All accepted applications, are then transferred to the Trainee view. In this view the lead trainer, trainers,discussion leaders, lesson maintainers can track progress of the trainees, provide feedback and update the status of every stage. They are also notified here if the Trainee has requested a discussion session.
+This list can be filtered on the following criteria:
+  * instructor assigned.
+  * start date of training.
+  * location of the training.
+
+  When the training is completed and the trainee has been certified, the lead instructor can export the trainee as instructor, to move their accounts to the Person table, and remove them from the Trainee table.
+![HTML MOCKUP ALL_TRAINEE](https://cloud.githubusercontent.com/assets/15982349/13816909/3401c4a4-ebb6-11e5-9799-0eb21ccd5eea.png)
+4. *Trainee Home page* -
+After getting accepted for the training program, the trainees recieve a randomly generated password, which they can use to log in to AMY, a trainee's home page is different from the AMY DASHBOARD, the AMY DASHBOARD is reservered for administrative purposes only. After logging in the trainees are directed to TRAINEE_HOME. This view will have a nav-bar - **HOME** , **ABOUT** and **NOTIFICATION**.
+  * *HOME*  shows the trainees progress, through a progress bar indicating which stages have been completed and the ones left. In this view the trainee can also request a discussion session by filling in a webform, with fields such as TOPIC, PREFERRED TIME, and so on.
```

Comment:
The login would make it easier for the trainees to track their progress through the program. They would know what the next step is, instead of writing emails back and forth to know what they should be doing next.